hera_cal/io.py

```diff
@@ -1184,6 +1185,8 @@ def load_vis(input_data, return_meta=False, filetype='miriad', pop_autos=False,
         pick_data_ants: boolean, if True and return_meta=True, return only antennas in data
         nested_dict: boolean, if True replace DataContainers with the legacy nested dictionary filetype
             where visibilities and flags are accessed as data[(0,1)]['nn']
+        return_nsamples: boolean, if True, return nsamples data container.
```

Comment:
I don't have any problem with adding this kwarg, per se, but `load_vis` is a really outdated use pattern that we mostly keep around to support old code. Maybe one day we'll deprecate it. Using `HERAData` is the more modern approach and the easier one to maintain.

So all that said, it appears that you're only updating this to support unit tests. I'd probably prefer leaving the old code as is and re-rewring new unit tests with HERAData. Thoughts?

Reply:
i've reverted this.

Reply:
I've replaced the pattern.